packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -611,13 +632,21 @@ function getActiveIndex(activeIds: string[], highlightedList: any[]): number {
 
 function getActiveElement(activeIds: string[], itemList: any[]): any {
   return itemList.find(item => {
-    if (item.allAttributeIds) {
+    if ('allAttributeIds' in item) {
       return isEqual(item.allAttributeIds.sort(), activeIds.sort());
+    } else if (isEqualId(item, activeIds)) {
+      return true;
+    } else if ('attributes' in item) {
+      return item.attributes.find((attribute: any) => isEqualId(attribute, activeIds));
     }
-    return isEqual([getId(item)].sort(), activeIds.sort());
+    return false;
   });
 }
 
+function isEqualId(item: any, id: string[]): boolean {
+  return isEqual([getId(item)].sort(), id.sort());
```

Comment:
This seems odd: does `getId` return a single ID? If so, why call `sort()` on a singleton array?